test/e2e/flowaggregator_test.go

```diff
@@ -514,6 +514,7 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 		t.Errorf("Error when running iperf3 client: %v", err)
 	}
 	bwSlice, srcPort := getBandwidthAndSourcePort(stdout)
+	require.Equal(t, 2, len(bwSlice), "bandwidth value and bandwidth unit are not available")
```

Comment:
I'd rather have this in a separate PR, even if it's just a one-line change. This is likely that we will be cherry-picking the iptables fix and this may create merge conflicts for no good reason.

Reply:
Indeed. Have removed the code. Will pay more attention next time.